CMR/python/test/cmr/search/test_common.py

```diff
@@ -185,12 +185,14 @@ def test_cmr_query_url(self):
     @patch('urllib.request.urlopen')
     def test_scroll(self, urlopen_mock):
         """ Test the scroll clear function to see if it returns an error or not"""
-        recorded_data_file = 'test/data/cmr/common/scroll_good.json'
+        recorded_data_file = os.path.join (os.path.dirname (__file__),
```

Comment:
Using `os.path.dirname (__file__)` resolves the resource location problem when running coverage from anywhere other than the `CMR/python` directory.